tests/dbus/test_dbus_client.cpp

```diff
@@ -349,6 +349,59 @@ void CheckTelemetryData(ThreadApiDBus *aApi)
 #if OTBR_ENABLE_LINK_METRICS_TELEMETRY
     TEST_ASSERT(telemetryData.low_power_metrics().link_metrics_entries_size() >= 0);
 #endif
+#if OTBR_ENABLE_BORDER_AGENT
+    TEST_ASSERT(telemetryData.wpan_border_router().border_agent_info().epskc_state() ==
```

Comment:
add test for activation scenario using ActivateEphemeralKeyMode/DeactivateEphemeralKeyMode dbus api?

Reply:
I'm planning to extend the test in `tests/scripts/thread-cert/border_router/test_ephemeral_key_counters.py` with dbus apis